src/app/scenario/scenario.component.html

```diff
@@ -29,10 +29,28 @@ <h3>Scenarios</h3>
 </div>
 <div class="clr-row">
   <div class="clr-col-4">
-    <clr-datagrid [clrDgSingleSelected]="unusedSelectedScenario" (clrDgSingleSelectedChange)="editScenario($event)" [clrDisableSelection]="!selectRbac">
+
+    <clr-dropdown>
+      <button class="dropdown-toggle" clrDropdownTrigger>
+        <span *ngIf="category==''">All</span>
```

Comment:
After selecting "All" or "No Category" the dropdown doesn't close by itself as it does with the other selectable categories. Tested with Firefox and Chrome.